alert-common/src/main/java/com/synopsys/integration/alert/common/provider/lifecycle/ProviderTask.java

```diff
@@ -62,10 +62,6 @@ public String getTaskName() {
         return ProviderTask.computeProviderTaskName(providerKey, getProviderProperties().getConfigId(), getClass());
     }
 
-    public final void setProviderPropertiesForRun(ProviderProperties providerProperties) {
-        this.providerProperties = providerProperties;
-    }
-
     protected ProviderProperties getProviderProperties() {
```

Comment:
I think `validateProviderProperties` and `invalidateProviderProperties` should also be deleted now.